packaging/ios/build.sh

```diff
@@ -44,10 +44,10 @@ fi
 
 # Which versions are we building against?
 SQLITE_VERSION=3090200
-LIBXML2_VERSION=2.9.2
-LIBXSLT_VERSION=1.1.28
-LIBZIP_VERSION=0.11.2
-LIBGIT2_VERSION=0.26.0
+#LIBXML2_VERSION=2.9.2
+LIBXSLT_VERSION=1.1.32
+LIBZIP_VERSION=1.2.0
+LIBGIT2_VERSION=v0.26.0
 
```

Comment:
hm, i still see the version changes here.

> +#LIBXML2_VERSION=2.9.2

not using a specific version and cloning the master would mean that we build the latest HEAD.
is my assumption correct?
see [1].

>+LIBXSLT_VERSION=1.1.32
>+LIBZIP_VERSION=1.2.0

these are technically version bumps. did you have troubles building with the older libraries?
LIBZIP 1.2.0 is a nice bump everywhere for the sake of using a 1.x release (they started using proper versioning, perhaps after i suggested it to them).
not sure about the need to bump LIBXSLT.

> +LIBGIT2_VERSION=v0.26.0

this isn't a version bump but it kind of stands out because of the extra `v`. the tag checkout is correct.
not a big issue.